pkg/providers/vsphere/virtualmachine/backup_test.go

```diff
@@ -234,6 +237,7 @@ func backupTests() {
 						}
 
 						Expect(virtualmachine.BackupVirtualMachine(backupOpts)).To(Succeed())
+						// getExpectedBackupObjectYaml empties the expected VM yaml's status
```

Comment:
Do you want to add an expect matcher for this to confirm that the condition does not exist?

Reply:
can do. yep

Reply:
hmm.. looked at it and this is probably the max we can do with the string matching that's already happening in verifyBackupDataInExtraConfig unless we unmarshal the backupData back to the VM object (which feels like overkill) 

Let me explain. 

So the expected yaml doesn't have a status (ie no backup condition too). The expect string matcher comparing the decoded backupData and expected Yaml (string) is in verifyBackupDataInExtraConfig (ie) no condition is present.